apps/src/blockly/addons/cdoSerializationHelpers.js

```diff
@@ -176,3 +262,28 @@ function reorderBlocks(blocks, blockOrderMap) {
 
   return orderedBlocks;
 }
+
+export function getCombinedSerialization(
+  mainWorkspaceSerialization,
+  hiddenWorkspaceSerialization
+) {
+  if (
```

Comment:
nit: add a comment explaining it's not possible to have a `hiddenWorkspaceSerialization` but no `mainWorkspaceSerialization`?